src/configs/singularity.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct SingularityConfig<'a> {
+    pub symbol: Option<SegmentConfig<'a>>,
+    pub label: &'a str,
+    pub prefix: &'a str,
+    pub suffix: &'a str,
+    pub style: Style,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for SingularityConfig<'a> {
+    fn new() -> Self {
+        SingularityConfig {
+            symbol: None,
+            label: "",
+            prefix: "[",
+            suffix: "]",
+            style: Color::Blue.bold().dimmed(),
+            disabled: true,
```

Comment:
You need to explicitly add code to `modules/singularity.rs` to disable the module. Currently there is no high order approach for starship to judge the disabled option when the module is disabled as a default.

Reply:
Done.